docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -1139,6 +1139,34 @@ The limit protects the system’s stability from potential abuse or mistakes, an
 
 > **Note**: Invalid metrics metadata are skipped during the ingestion, and valid metadata within the same request are ingested.
 
+### err-mimir-distributor-max-ingestion-rate
+
+This critical error occurs when the rate of received samples, exemplars and metadata per second is exceeded in a distributor.
+
+The distributor implements a rate limit on the samples per second that can be ingested, and it's used to protect a distributor from overloading in case of high traffic.
+The limit is a per-instance limit and it's applied on all samples, exemplars and metadata received, across all tenants, in each distributor.
+
+How to **fix** it:
+
+- Scale up distributors.
+- Increase the limit by using the `-distributor.instance-limits.max-ingestion-rate` option.
+
+### err-mimir-distributor-max-inflight-push-requests
+
+This error occurs when a distributor rejects a write request because the max inflight requests limit has been reached.
+
+How it **works**:
+
+- The distributor has per-instance limit on the number of inflight write (push) requests.
+- The limit applies on all inflight write requests, across all tenants, and is used to protect the distributor from overloading in case of high traffic.
```

Comment:
```suggestion
- The limit applies to all in-flight write requests, across all tenants, and it protects the distributor from becoming overloaded in case of high traffic.
```

Reply:
I am a bit torn on "applies to" AKA "relates to" versus "is applied to", in which case what is doing the _applying_?

Reply:
In this context, what we want to say is that the limit spans in-flight requests across all of the tenants within each distributor.